ov/profiles.go

```diff
@@ -409,9 +409,11 @@ func (c *OVClient) DeleteProfile(name string) error {
 
 		// submit delete task
 		t, err := c.SubmitDeleteProfile(profile)
-		err = t.Wait()
-		if err != nil {
-			return err
+		if c.APIVersion < 1000 {
```

Comment:
What about this resource. How do you know when this gets complete?

Reply:
Since it doesn't receive a task in response if the operation fails Oneview will return an error.